### PR TITLE
AM1 7.1 new address HI telephone capture

### PR DESCRIPTION
**Pull request by AdamHawtin**
# Motivation and Context
Individual telephone capture can be requested on newly reported skeleton cases. Also a quality of life improvement with smoke test tags and target to quickly smoke test an environment

# What has changed
* New scenerio for individual telephone capture on new address
* Smoke test tags and run target
* Update README

# How to test?
Run with linked case processor branch
Run `make smoke_test`, it should complete in under 1:30min

# Links
https://trello.com/c/zwSsVjsD/713-am1-part-71-new-address-create-hi-case-for-telephone-capture-3
https://github.com/ONSdigital/census-rm-case-processor/pull/149

# Screenshots (if appropriate):

### CODE REVIEW

**Comment by AdamHawtin** [comment]
[AM1 part 7.1 - New Address - Create HI Case for telephone capture (3)](https://trello.com/c/zwSsVjsD/713-am1-part-71-new-address-create-hi-case-for-telephone-capture-3)

File: acceptance_tests/features/ad_hoc_uac_qid.feature [hunk 1]
```diff
@@ -5,9 +5,13 @@ Feature: A UAC/QID pair can be requested for a case
     When a UAC/QID pair is requested with questionnaire type "<questionnaire type>"
     Then a UAC updated message with "<questionnaire type>" questionnaire type is emitted
 
+    @smoke
```
Comment (NickGrantX) [inline acceptance_tests/features/ad_hoc_uac_qid.feature:8]: wouldn't you normally tag the scenario, not the examples?

**Review by NickGrantX — COMMENTED** [review]
Not sure I understand why you put `@smoke` on the examples not on the scenarios

File: acceptance_tests/features/ad_hoc_uac_qid.feature (hunk 1 above)
Comment (AdamHawtin) [reply acceptance_tests/features/ad_hoc_uac_qid.feature:8]: @NickGrantONS The tag on the scenario would run all the examples, tags on the example are treated as tags on only that example's scenario when it expands the n examples into n scenarios. I was aiming to just tag a variety of features that would run quickly but touch most areas of RM, running multiple examples for these scenarios didn't seem to add much to the variety but adds a lot to the execution time.

**Review by gavinleeedwards — APPROVED** [review]
Tested running all tests against the case processor branch then just the smoke tests.  All looking good 👍

File: acceptance_tests/features/ad_hoc_uac_qid.feature (hunk 1 above)
Comment (NickGrantX) [reply acceptance_tests/features/ad_hoc_uac_qid.feature:8]: ah I see what you mean

**Review by NickGrantX — APPROVED** [review]
LGTM 👍

**Pull request merged**